packages/kbn-i18n/README.md

```diff
@@ -296,11 +296,32 @@ import { Header } from './components/header';
 module.directive('headerGlobalNav', (reactDirective) => {
   return reactDirective(injectI18nProvider(Header));
 });
-
 ```
 
 **NOTE:** To minimize the chance of having multiple `I18nProvider` components in the React tree, try to use `injectI18nProvider` or `I18nProvider` only to wrap the topmost component that you render, e.g. the one that's passed to `reactDirective` or `ReactDOM.render`.
 
+### FormattedRelative
+
+[React-intl readme](https://github.com/yahoo/react-intl/wiki/Components#formattedrelative)\
+`i18n-locale` config variable is used for translation.\
```

Comment:
I'd rewrite this `[React-intl readme](https://github.com/yahoo/react-intl/wiki/Components#formattedrelative) 
`i18n-locale` config variable is used for translation. formats field should be one of 'years' | 'months' | 'days' | 'hours' | 'minutes' | 'seconds'.` with:
```
`FormattedRelative` excepts several attributes (read more [here](https://github.com/yahoo/react-intl/wiki/Components#formattedrelative)), including 

- `value` that can be parsed as a date,
- `formats` that should be one of `'years' | 'months' | 'days' | 'hours' | 'minutes' | 'seconds'` (this options are configured in formats.ts (link to relative in formats.ts))
-  etc.

```